src/pages/Login.tsx

```diff
@@ -63,8 +91,12 @@ export default () => {
 		<Wrap>
 			<Login>
 				<Logo width={100} />
-				<h1 style={{fontFamily:'courier'}}>idena-pocket</h1>
+				<h1 className="title">Idena-pocket</h1>
+				<p className="description extrapadding">Web-wallet for Idena</p>
+
 				{!storage.encryptedSeed && <Button to="import-mnemonic" text="Import mnemonic" />}
+				<span className="greyback"><p className="description">Or</p></span>
```

Comment:
The "or" `<p>` should not be displayed when user imported a key:
https://www.dropbox.com/s/zwprc4rnmhtjbme/idena-pocket-issue-01-login-page.png?dl=0

Something like this:
```
{!storage.encryptedSeed && <span className="greyback"><p className="description">Or</p></span>}
```

Reply:
All done. Converted the or segment into a component instead of inline HTML.